### ISSUE TITLE
Error: Minified React error #143 on Production Build

### ISSUE THREAD

**Issue by jcald1** [reactjs/redux#2758]
Running Redux v3.7.2 on Ubuntu 16.04.03 with the following redux example project: redux/examples/real-world   The development version works fine for me.  For the production build:

1) export NODE_ENV=production
2) npm run build
3) cd build
4) edit index.html.  Change "/static/js/main.5b96d134.js" to "./static/js/main.5b96d134.js"
5) Open browser link to the index.html file.
6) I get the following error:

```
react-dom.production.min.js:164 Error: Minified React error #143; visit http://facebook.github.io/react/docs/error-decoder.html?invariant=143 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at r (react.production.min.js:11)
    at Object.only (react.production.min.js:19)
    at n.render (Provider.js:49)
    at c (react-dom.production.min.js:130)
    at beginWork (react-dom.production.min.js:133)
    at o (react-dom.production.min.js:161)
    at a (react-dom.production.min.js:161)
    at u (react-dom.production.min.js:162)
    at x (react-dom.production.min.js:169)
    at w (react-dom.production.min.js:168)
```
When I follow the same steps for the "redux/examples/async" code, I don't get any errors.

**Comment by timdorr** [comment]
This is a bug tracker, not a support system. For usage questions, please use Stack Overflow or Reactiflux where there are a lot more people ready to help you out. Thanks!

**Comment by markerikson** [comment]
Eh, it is actually sort of a bug report - the complaint is about our own "Real World" example.

**Comment by jcald1** [comment]
So can we re-open this since the issue is with the redux examples code?

**Comment by markerikson** [comment]
On Windows 10, building the example and running it works fine for me, no warnings or errors in the page.

I also don't understand why you needed to manually set `NODE_ENV=production`, or edit index.html in the build output.

**Comment by jcald1** [comment]
I just set that variable in case it was needed by the build script.  I'll just skip that from now on.  As for the index.html change, that's because without the "homepage" package.json setting, the site is assumed to be at the root.   I went ahead and just rebuilt without modifying anything and copied the build files to my root directory and I still get the error.  

Did you both build and run on Windows 10?
